{
	"id": "AVG-574",
	"advisories": [
		{
			"id": "AVG-574",
			"severity": [
				{
					"type": "vendor",
					"source": "security.archlinux.org",
					"vendor": "High"
				}
			],
			"references": [
				{
					"source": "security.archlinux.org",
					"url": "https://security.archlinux.org/AVG-574"
				},
				{
					"source": "security.archlinux.org",
					"url": "https://bugs.archlinux.org/task/56832"
				}
			]
		},
		{
			"id": "ASA-201801-4",
			"references": [
				{
					"source": "security.archlinux.org",
					"url": "https://security.archlinux.org/ASA-201801-4"
				}
			]
		}
	],
	"vulnerabilities": [
		{
			"id": "CVE-2017-8824",
			"references": [
				{
					"source": "security.archlinux.org",
					"url": "https://security.archlinux.org/CVE-2017-8824"
				}
			]
		},
		{
			"id": "CVE-2017-5754",
			"references": [
				{
					"source": "security.archlinux.org",
					"url": "https://security.archlinux.org/CVE-2017-5754"
				}
			]
		},
		{
			"id": "CVE-2017-17864",
			"references": [
				{
					"source": "security.archlinux.org",
					"url": "https://security.archlinux.org/CVE-2017-17864"
				}
			]
		},
		{
			"id": "CVE-2017-17863",
			"references": [
				{
					"source": "security.archlinux.org",
					"url": "https://security.archlinux.org/CVE-2017-17863"
				}
			]
		},
		{
			"id": "CVE-2017-17862",
			"references": [
				{
					"source": "security.archlinux.org",
					"url": "https://security.archlinux.org/CVE-2017-17862"
				}
			]
		},
		{
			"id": "CVE-2017-17857",
			"references": [
				{
					"source": "security.archlinux.org",
					"url": "https://security.archlinux.org/CVE-2017-17857"
				}
			]
		},
		{
			"id": "CVE-2017-17856",
			"references": [
				{
					"source": "security.archlinux.org",
					"url": "https://security.archlinux.org/CVE-2017-17856"
				}
			]
		},
		{
			"id": "CVE-2017-17855",
			"references": [
				{
					"source": "security.archlinux.org",
					"url": "https://security.archlinux.org/CVE-2017-17855"
				}
			]
		},
		{
			"id": "CVE-2017-17854",
			"references": [
				{
					"source": "security.archlinux.org",
					"url": "https://security.archlinux.org/CVE-2017-17854"
				}
			]
		},
		{
			"id": "CVE-2017-17853",
			"references": [
				{
					"source": "security.archlinux.org",
					"url": "https://security.archlinux.org/CVE-2017-17853"
				}
			]
		},
		{
			"id": "CVE-2017-17852",
			"references": [
				{
					"source": "security.archlinux.org",
					"url": "https://security.archlinux.org/CVE-2017-17852"
				}
			]
		},
		{
			"id": "CVE-2017-17806",
			"references": [
				{
					"source": "security.archlinux.org",
					"url": "https://security.archlinux.org/CVE-2017-17806"
				}
			]
		},
		{
			"id": "CVE-2017-17805",
			"references": [
				{
					"source": "security.archlinux.org",
					"url": "https://security.archlinux.org/CVE-2017-17805"
				}
			]
		},
		{
			"id": "CVE-2017-17741",
			"references": [
				{
					"source": "security.archlinux.org",
					"url": "https://security.archlinux.org/CVE-2017-17741"
				}
			]
		},
		{
			"id": "CVE-2017-17712",
			"references": [
				{
					"source": "security.archlinux.org",
					"url": "https://security.archlinux.org/CVE-2017-17712"
				}
			]
		},
		{
			"id": "CVE-2017-17558",
			"references": [
				{
					"source": "security.archlinux.org",
					"url": "https://security.archlinux.org/CVE-2017-17558"
				}
			]
		},
		{
			"id": "CVE-2017-17450",
			"references": [
				{
					"source": "security.archlinux.org",
					"url": "https://security.archlinux.org/CVE-2017-17450"
				}
			]
		},
		{
			"id": "CVE-2017-17449",
			"references": [
				{
					"source": "security.archlinux.org",
					"url": "https://security.archlinux.org/CVE-2017-17449"
				}
			]
		},
		{
			"id": "CVE-2017-17448",
			"references": [
				{
					"source": "security.archlinux.org",
					"url": "https://security.archlinux.org/CVE-2017-17448"
				}
			]
		},
		{
			"id": "CVE-2017-16996",
			"references": [
				{
					"source": "security.archlinux.org",
					"url": "https://security.archlinux.org/CVE-2017-16996"
				}
			]
		},
		{
			"id": "CVE-2017-16995",
			"references": [
				{
					"source": "security.archlinux.org",
					"url": "https://security.archlinux.org/CVE-2017-16995"
				}
			]
		}
	],
	"detection": [
		{
			"ecosystem": "arch",
			"vulnerable": true,
			"package": {
				"name": "linux-hardened"
			},
			"affected": {
				"type": "pacman",
				"range": [
					{
						"lt": "4.14.11.a-1"
					}
				],
				"fixed": [
					"4.14.11.a-1"
				]
			}
		}
	]
}
